CMakeLists.txt

```diff
@@ -62,9 +62,13 @@ target_link_libraries(TestEntGen EntityLib)
 add_test(TestEntGen TestEntGen)
 
 file(GLOB TESTENTITYLIB_SOURCES "Test/*.cpp" "Test/*.h" "Test/*.hpp")
-add_executable(TestEntityDLL ${TESTENTITYLIB_SOURCES})
-target_include_directories(TestEntityDLL PRIVATE ${CMAKE_CURRENT_BINARY_DIR}/EntGen/cpp2)
-target_include_directories(TestEntityDLL PRIVATE submodules/cxxopts/include)
-target_include_directories(TestEntityDLL PRIVATE external/catch)
-target_link_libraries(TestEntityDLL EntityLib)
-add_test(TestEntityDLL TestEntityDLL)
+add_executable(UnitTest ${TESTENTITYLIB_SOURCES})
+target_include_directories(UnitTest PRIVATE ${CMAKE_CURRENT_BINARY_DIR}/EntGen/cpp2)
+target_include_directories(UnitTest PRIVATE submodules/cxxopts/include)
+target_include_directories(UnitTest PRIVATE external/catch)
+target_link_libraries(UnitTest EntityLib)
+
+file(GLOB UPDATESCHEMA_SOURCES "UpdateSchemas/*.cpp" "UpdateSchemas/*.h" "UpdateSchemas/*.hpp")
+add_executable(UpdateSchemas ${UPDATESCHEMA_SOURCES})
+target_include_directories(UpdateSchemas PRIVATE submodules/cxxopts/include)
+target_link_libraries(UpdateSchemas EntityLib)
```

Comment:
This seems to be a copy/paste of the code at line 45-48.

Reply:
let me check it out .